App.js

```diff
@@ -1,12 +1,10 @@
-import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
 export default function App() {
   return (
     <View style={styles.container}>
       <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
```

Comment:
Nothing wrong with the code. But suggest, moving forward, we have the content of the PR match the title.
However this is a very small change to App.js and is very early in development. Therefore I personally would not stand in the way of this PR being approved.